### PR TITLE
cypress test delete collection version

### PR DESCRIPTION
**Pull request by ShaiahWren**
Issue: https://issues.redhat.com/browse/AAH-1008

Assignment: Cypress test for deleting a collection version. 

Note: I tried putting a `data-cy` attribute on the StatefulDropdown in /collection-header.tsx but it does not appear in the file tree. So I used a className instead, for now...

### CODE REVIEW

**Comment by himdel** [comment]
Heads up re test failure.. now that galaxykit is updated (`python -m pip install --user galaxykit --upgrade`, 0.5.2+), `collection upload` works .. except it ends up still needing approval.
That will go away once https://github.com/ansible/ansible-hub-ui/pull/1082 is merged, or you can add https://github.com/ansible/ansible-hub-ui/pull/1082/commits/29db1a4c9b526c2896fe3c213beffd347a9dbd35 to get tests green :)

File: test/cypress/integration/collection.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  it('deletes a collection version', () => {
+    // cy.galaxykit('-i namespace create', 'testns1');
+    cy.galaxykit('-i collection upload my_namespace my_collection');
+    cy.menuGo('Collections > Collections');
+    cy.get('a[href*="ui/repo/published/my_namespace/my_collection"]').click();
+    cy.get('[data-cy=kebab-toggle]').click();
+    cy.get('[data-cy=delete-version-dropdown]').click();
+    cy.get('input[id=delete_confirm]').click();
+    cy.get('button').contains('Delete').click();
+    cy.wait(5000);
```
Comment (himdel) [inline test/cypress/integration/collection.js:18]: https://github.com/ansible/ansible-hub-ui/pull/1117#issuecomment-951721515

File: .github/workflows/cypress.yml [hunk 2]
```diff
@@ -86,6 +86,7 @@ jobs:
           RH_ENTITLEMENT_REQUIRED='insights'
           TOKEN_AUTH_DISABLED=True
           X_PULP_CONTENT_HOST='localhost'
+          GALAXY_REQUIRE_CONTENT_APPROVAL=False
```
Comment (himdel) [inline .github/workflows/cypress.yml:89]: (from #1082)

**Comment by ShaiahWren** [comment]
@himdel so... I rebased, and now the two tests pass individually ( it.only...) but not together. I see an error 403.
Screen Shot 2021-11-03 at 11 30 21 PM

**Comment by himdel** [comment]
@ShaiahWren ah, that's a fun one ... `before` -> `beforeEach` should fix it :)

`cy.login` only happens in `before` in that test now, so it logs in for just one test, and forgets about that once that test ends, so the next one is no longer logged in :) `beforeEach` happens before each test, so both will be logged in :)

**Comment by ShaiahWren** [comment]
Ah okay, perfect!
I am out of the office today and away from my laptop. Would you mind making
that small change and then merging the pr for me? I would really appreciate
it. The other 4.4 items on Jira should be already merged, if they ask in
the meeting :)

**Comment by himdel** [comment]
Sure, no problem, added, will merge when green :)

**Comment by ShaiahWren** [comment]
Thanks a million 🙏

**Pull request merged**